src/components/ImageWithSizeCalculation.js

```diff
@@ -29,6 +30,14 @@ const defaultProps = {
  * it can be appropriately resized.
  */
 class ImageWithSizeCalculation extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            loading: true,
+        }
```

Comment:
@mdneyazahmad Thank you for the PR! Got one styling mistake here.

Reply:
Fixed the linting error

Reply:
Thanks for the bump Vit :)